applications/endoscopy_tool_tracking/cpp/endoscopy_tool_tracking.yaml

```diff
@@ -26,7 +26,8 @@ extensions:
 # Uncomment the following extension when using yuan as source
 #  - gxf_extensions/yuan_qcap/libgxf_qcap_source.so
 
-source: "replayer" # "replayer" or "aja" or "deltacast" or "yuan"
+source: "replayer"   # "replayer" or "aja" or "deltacast" or "yuan"
+visualizer: "vtk"    # "vtk" or "holoviz"
```

Comment:
Keep default to holoviz and in the readme add step in build instructions to run the `sed` command. Also provide `sed` command to switch back.